python/pyiceberg/io/pyarrow.py

```diff
@@ -239,3 +266,102 @@ def delete(self, location: Union[str, InputFile, OutputFile]) -> None:
             elif e.errno == 13 or "AWS Error [code 15]" in str(e):
                 raise PermissionError(f"Cannot delete file, access denied: {location}") from e
             raise  # pragma: no cover - If some other kind of OSError, raise the raw error
+
+
+def convert_iceberg_schema_to_pyarrow(schema: Schema) -> pa.schema:
+    return visit(schema, _ConvertToArrowSchema())
```

Comment:
Overall, conversion looks correct to me.

This method name is a bit long. How about `schema_to_pyarrow`?

Reply:
I like it, thanks!